### ISSUE TITLE
Don’t disable trackpad if only mouse available is touchscreen

### ISSUE THREAD

**Issue by usr-sse2** [acidanthera/bugtracker#452]
https://github.com/alexandred/VoodooI2C/issues/147

**Comment by vit9696** [comment]
@usr-sse2 can you clarify why is that an error? I am afraid that we may not even have control over this, as it should be decided on macOS side, not on the driver side.

**Comment by usr-sse2** [comment]
@vit9696 No, the control is on our side. See `ApplePS2SynapticsTouchPad::notificationHIDAttachedHandlerGated`. (But it may be the case that currently both we and macOS are disabling the trackpad – I haven't checked it. But when VoodooPS2 worked not through MT2 simulation, the system didn't control it).

This is an error because the semantics of this setting is to disable built-in pointing devices when an external pointing device is connected. Macs don't have built-in touchscreens, so for Macs it's all OK. Lenovo Yoga has a built-in touchscreen, so the checkbox becomes useless because it just disables the trackpad always.

**Comment by usr-sse2** [comment]
I checked, we are already checking the device class, and for touchscreen we're not disabling the trackpad, so macOS does it. Still the code in VoodooPS2 shouldn't be disabled, because it also affects physical buttons and trackpoint, which are sent in bypass of MT2 simulator.
